### PR TITLE
Update docs

### PR DESCRIPTION
**Pull request by kimlisa**
#### Definition:
admin-guide.md
- add section on how to remove a node from cluster

architecture_overview.md
- fix a broken link to quickstart

trustedclusters.md
- remove confusing phrase
- add link to how to use a cluster join token

### CODE REVIEW

**Review by benarent — APPROVED** [review]

**Review by stevenGravy — COMMENTED** [review]
I'd add that you can forcibly remove a node with `tctl rm nodes/<uuid>` after getting the UUID from tctl nodes ls.

File: docs/4.2/admin-guide.md [hunk 1]
```diff
@@ -858,6 +858,41 @@ turing        d52527f9-b260-41d0-bb5a-e23b0cfe0f8f     10.1.0.5:3022      distro
...
+### Removing Nodes from a Cluster
+
+To remove a node from a cluster, simply stop the node that you want to remove. 
+
+#### TTL time
+Node processes that are stopped have a TTL time of 10 minutes.
+You can restart the node within this time limit to keep the node. 
+
+After 10 minutes, the node is no longer considered `active` and is removed from 
+the `active SSH nodes` list within the cluster. You can  check for removable with
```
Comment (webvictim) [inline docs/4.2/admin-guide.md]: Double space in `can  check`

Also, this doesn't quite scan. Should be `You can check for removable nodes nodes with this command` maybe?

File: docs/4.2/admin-guide.md [hunk 2]
```diff
@@ -858,6 +858,41 @@ turing        d52527f9-b260-41d0-bb5a-e23b0cfe0f8f     10.1.0.5:3022      distro
...
+# STOP the node process first
+
+# get the UUID of the node you want to remove
+$ tctl nodes ls
+Nodename      UUID                                     Address            Labels
+---------     -------                                  -------            ------
+turing        d52527f9-b260-41d0-bb5a-e23b0cfe0f8f     10.1.0.5:3022      distro:ubuntu
+dijkstra      c9s93fd9-3333-91d3-9999-c9s93fd98f43     10.1.0.6:3022      distro:debian
+
+# forefully remove the node
```
Comment (webvictim) [inline docs/4.2/admin-guide.md]: forcefully

File: docs/4.2/admin-guide.md [hunk 3]
```diff
@@ -858,6 +858,41 @@ turing        d52527f9-b260-41d0-bb5a-e23b0cfe0f8f     10.1.0.5:3022      distro
...
+
+#### TTL time
+Node processes that are stopped have a TTL time of 10 minutes.
+You can restart the node within this time limit to keep the node. 
+
+After 10 minutes, the node is no longer considered `active` and is removed from 
+the `active SSH nodes` list within the cluster. You can  check for removable with 
+the command: [`tctl nodes ls`](cli-docs.md#tctl-nodes-ls).
+
+#### Remove a Node Immediateley
```
Comment (webvictim) [inline docs/4.2/admin-guide.md]: immediately

File: docs/4.2/admin-guide.md [hunk 4]
```diff
@@ -858,6 +858,41 @@ turing        d52527f9-b260-41d0-bb5a-e23b0cfe0f8f     10.1.0.5:3022      distro
 dijkstra      c9s93fd9-3333-91d3-9999-c9s93fd98f43     10.1.0.6:3022      distro:debian
 ```
 
+### Removing Nodes from a Cluster
+
+To remove a node from a cluster, simply stop the node that you want to remove. 
+
+#### TTL time
```
Comment (webvictim) [inline docs/4.2/admin-guide.md]: Technically TTL is time-to-live to "time-to-live time" isn't quite right.

I'd just change this to "Time to live"

File: docs/4.2/admin-guide.md [hunk 5]
```diff
@@ -858,6 +858,41 @@ turing        d52527f9-b260-41d0-bb5a-e23b0cfe0f8f     10.1.0.5:3022      distro
...
+Node processes that are stopped have a TTL time of 10 minutes.
```
Comment (webvictim) [inline docs/4.2/admin-guide.md]: TTL time -> time to live

File: docs/4.2/admin-guide.md (hunk 1 above)
Comment (kimlisa) [reply docs/4.2/admin-guide.md]: Removed double space. I reworded it to `You can check that the node has been removed with
the command`. What do you think?

File: docs/4.2/admin-guide.md (hunk 2 above)
Comment (kimlisa) [reply docs/4.2/admin-guide.md]: revised

File: docs/4.2/admin-guide.md (hunk 3 above)
Comment (kimlisa) [reply docs/4.2/admin-guide.md]: revised

File: docs/4.2/admin-guide.md (hunk 4 above)
Comment (kimlisa) [reply docs/4.2/admin-guide.md]: revised

**Review by webvictim — APPROVED** [review]

**Pull request merged**